inst/qml/auditBayesianWorkflow.qml

```diff
@@ -16,454 +16,871 @@
 // When making changes to this file always mention @koenderks as a
 // reviewer in the Pull Request
 
-import QtQuick			2.8
-import QtQuick.Layouts	1.3
-import JASP.Controls	1.0
-import JASP.Widgets		1.0
+import QtQuick										2.8
+import QtQuick.Layouts								1.3
+import JASP.Controls								1.0
+import JASP.Widgets									1.0
 
+// --------------------------------------------------------------------------------------------------------------------------------------------
+// -------------------------------------------------  BEGIN WORKFLOW  -------------------------------------------------------------------------
+// --------------------------------------------------------------------------------------------------------------------------------------------
 
 Form
 {
-	
-	columns: 			1
 
-		// Extra options
-		CheckBox { name: "workflow"; checked: true; visible: false}
-		CheckBox { name: "useSumStats"; checked: false; visible: false}
+	columns: 										1
+
+	// Extra options to be included without being visible
+	CheckBox 
+	{ 
+		name: 										"workflow"
+		checked: 									true
+		visible: 									false 
+	}
+
+	CheckBox 
+	{ 
+		name: 										"bayesianAnalysis"
+		checked: 									true
+		visible: 									false 
+	}
+
+	CheckBox 
+	{ 
+		name: 										"useSumStats"
+		checked: 									false
+		visible: 									false 
+	}
 
 	// --------------------------------------------------------------------------------------------------------------------------------------------
 	// ---------------------------------------------------  PLANNING  -----------------------------------------------------------------------------
 	// --------------------------------------------------------------------------------------------------------------------------------------------
 
 	Section
 	{
-		id: 		planningPhase
-		text: 		planningPhase.expanded ? qsTr("<b>1. Planning</b>") : qsTr("1. Planning")
-		expanded:	!samplingChecked.checked
-		columns: 	1
+		id: 										planningPhase
+		text: 										planningPhase.expanded ? qsTr("<b>1. Planning</b>") : qsTr("1. Planning")
+		expanded:									!samplingChecked.checked
+		columns: 									1
 
 		GridLayout
 		{
-			columns: 2
-			enabled: !pasteVariables.checked
+			columns: 								3
 
-			RadioButtonGroup
+			GroupBox 
 			{
-				id: 	materiality
-				name: 	"materiality"
-				title: 	qsTr("Population Materiality")
+				title: 								"Sampling Objectives"
+				enabled: 							!pasteVariables.checked
+				columns:							2
 
-				RowLayout
+				CheckBox
 				{
-					RadioButton
+					id: 							performanceMateriality
+					text: 							"Test against a performance materiality"
```

Comment:
`qsTr`

Reply:
also, these sections seem to repeat across the qml files, maybe put them into a custom qml component?